avm/res/cache/redis-enterprise/tests/e2e/active-geo-replication/dependencies.bicep

```diff
@@ -0,0 +1,44 @@
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the geo-replication group.')
+param geoReplicationGroupName string
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'creagr'
```

Comment:
Should be replaced with `redisClusterName` and the `serviceShort` only be specified in the `main.test.bicep`. 

The idea: All names of resources that 'could' conflict with one another (usually top-level resources) should be generated in the `main.test.bicep` and be passed down to where they are needed. All other names / properties should be defined directly where they are used.

The main reason for this design decision is that it makes it quite easy to understand 1. what resources are deployed and 2. (when troubleshooting) which resource name of the error message links to which resource (as they're all visible in one place).

Reply:
Got it, could you please take another look at the latest changes?

Reply:
It looks like a dream come true @JeffreyCA 💪